internal/testrunner/runners/system/tester.go

```diff
@@ -421,6 +421,9 @@ func (r *tester) createAgentInfo(policy *kibana.Policy, config *testConfig, runI
 		info.Agent.User = "root"
 	}
 
+	// Set linux capabilities to the minimum set required by the agent
+	info.Agent.LinuxCapabilities = []string{"CHOWN", "SETFCAP", "SETPCAP"}
```

Comment:
This would overwrite the capabilities defined in the configuration file if any.

The settings from the config file are copied into the `info` variable here:
https://github.com/elastic/elastic-package/pull/1937/files#diff-0bdd696fb9abd5e4aa12a3bab634cbac31ec1a1349b3a61e8a784a6bdeb6e7d8R416

It would needed to merge both arrays/slices to not lose the values from the configuration file.